oeps/oep-0008.rst

```diff
@@ -0,0 +1,323 @@
+==================================
+OEP-8: Unit Testing Best Practices
+==================================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-8 </oeps/oep-0008>`             |
++===============+===========================================+
+| Title         | Unit Testing Best Practices               |
++---------------+-------------------------------------------+
+| Last Modified | 2016-09-09                                |
++---------------+-------------------------------------------+
+| Author        | Jesse Zoldak                              |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Best Practice                             |
++---------------+-------------------------------------------+
+| Created       | 2016-09-09                                |
++---------------+-------------------------------------------+
+| `Resolution`  |                                           |
++---------------+-------------------------------------------+
+| `Replaces`    |                                           |
++---------------+-------------------------------------------+
+| `Replaced-By` |                                           |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+The edX ecosystem has grown from a handful of developers working on a few
+repos to many developers working on many repos. As such, questions arise as to
+tools, frameworks, and patterns to use in unit testing. This proposal is
+intended to codify best practices and to provide a reference for contributors
+when making choices.
```

Comment:
That's a good point. I had imagined that future changes would be made in PRs to this OEP, and can include some wording to that effect.
Would that suffice, or were you getting at something different?